xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1635,11 +1635,11 @@ bool CGUIWindowVideoBase::GetDirectory(const std::string &strDirectory, CFileIte
     newPlaylist->m_bIsFolder = true;
     items.Add(newPlaylist);
 
-/*    newPlaylist.reset(new CFileItem("newplaylist://", false));
```

Comment:
It's working now. The prerequisites for this change to work are already merged in master.

I'll split it in two commits, but why should they be split in different PR? Adding the music video library to the music menu, but not the music library sounds like a half-done feature.